elm.json

```diff
@@ -1,79 +1,79 @@
-        "elm-community/maybe-extra": "5.2.0 <= v < 6.0.0",
-        "elm-community/string-extra": "4.0.0 <= v < 5.0.0",
-        "krisajenkins/remotedata": "6.0.0 <= v < 7.0.0",
-        "mgold/elm-nonempty-list": "4.1.0 <= v < 5.0.0",
-        "noahzgordon/elm-color-extra": "1.0.0 <= v < 2.0.0",
-        "rtfeldman/elm-iso8601-date-strings": "1.1.4 <= v < 2.0.0",
-        "stoeffel/set-extra": "1.2.0 <= v < 2.0.0",
-        "wernerdegroot/listzipper": "4.0.0 <= v < 5.0.0"
-    },
-    "test-dependencies": {
-        "elm-explorations/test": "1.2.0 <= v < 2.0.0"
-    }
+  "type": "package",
+  "name": "unison/ui-core",
+  "summary": "Core libraries and design system for the UIs of the Unison programming language",
+  "license": "MIT",
+  "version": "1.0.0",
+  "exposed-modules": [
+    "UI.AppHeader",
+    "UI.Color",
+    "UI.Sidebar",
+    "UI.Banner",
+    "UI.CopyField",
+    "UI.Toolbar",
+    "UI.Button",
+    "UI.Modal",
+    "UI.Tooltip",
+    "UI.Card",
+    "UI.FoldToggle",
+    "UI.PageLayout",
+    "UI.PageContent",
+    "UI.PageHeader",
+    "UI.Avatar",
+    "UI.Click",
+    "UI.Icon",
+    "UI.KeyboardShortcut",
+    "UI.KeyboardShortcut.Key",
+    "UI.KeyboardShortcut.KeyboardEvent",
+    "Code.CodebaseApi",
+    "Code.CodebaseTree",
+    "Code.Config",
+    "Code.Finder",
+    "Code.FullyQualifiedName",
+    "Code.FullyQualifiedNameSet",
+    "Code.Hash",
+    "Code.HashQualified",
+    "Code.Hashvatar",
+    "Code.Namespace",
+    "Code.Perspective",
+    "Code.Project",
+    "Code.Syntax",
+    "Code.UrlParsers",
+    "Code.Workspace",
+    "Lib.Util",
+    "Lib.OperatingSystem",
+    "Lib.SearchResults",
+    "Lib.TreePath",
+    "Lib.Color.Harmony",
+    "Lib.UnicodeSort",
+    "Lib.HttpApi"
+  ],
+  "elm-version": "0.19.0 <= v < 0.20.0",
+  "dependencies": {
+    "avh4/elm-color": "1.0.0 <= v < 2.0.0",
+    "danhandrea/elm-date-format": "2.0.1 <= v < 3.0.0",
+    "elm/browser": "1.0.0 <= v < 2.0.0",
+    "elm/core": "1.0.0 <= v < 2.0.0",
+    "elm/html": "1.0.0 <= v < 2.0.0",
+    "elm/http": "2.0.0 <= v < 3.0.0",
+    "elm/json": "1.1.0 <= v < 2.0.0",
+    "elm/parser": "1.1.0 <= v < 2.0.0",
+    "elm/regex": "1.0.0 <= v < 2.0.0",
+    "elm/svg": "1.0.0 <= v < 2.0.0",
+    "elm/url": "1.0.0 <= v < 2.0.0",
+    "elm-community/html-extra": "3.4.0 <= v < 4.0.0",
+    "elm-community/json-extra": "4.3.0 <= v < 5.0.0",
+    "elm-community/list-extra": "8.2.0 <= v < 9.0.0",
+    "elm-community/maybe-extra": "5.2.0 <= v < 6.0.0",
+    "elm-community/string-extra": "4.0.0 <= v < 5.0.0",
+    "krisajenkins/remotedata": "6.0.0 <= v < 7.0.0",
+    "mgold/elm-nonempty-list": "4.1.0 <= v < 5.0.0",
+    "noahzgordon/elm-color-extra": "1.0.0 <= v < 2.0.0",
+    "rtfeldman/elm-iso8601-date-strings": "1.1.4 <= v < 2.0.0",
+    "stoeffel/set-extra": "1.2.0 <= v < 2.0.0",
+    "wernerdegroot/listzipper": "4.0.0 <= v < 5.0.0"
+  },
+  "test-dependencies": {
+    "elm-explorations/test": "1.2.0 <= v < 2.0.0"
+  }
```

Comment:
Can we "unformat" this file? Elm manages it and will re-format on the next `elm install` I believe, which will add a bit of commit noise